packages/components/src/toolbar-dropdown-menu/index.js

```diff
@@ -10,33 +10,30 @@ import ToolbarItem from '../toolbar-item';
 import ToolbarContext from '../toolbar-context';
 import DropdownMenu from '../dropdown-menu';
 
-function ToolbarDropdownMenu( props ) {
+function ToolbarDropdownMenu( props, ref ) {
 	const accessibleToolbarState = useContext( ToolbarContext );
 
 	if ( ! accessibleToolbarState ) {
 		return <DropdownMenu { ...props } />;
 	}
 
 	// ToobarItem will pass all props to the render prop child, which will pass
-	// all props to the toggle of DrpodownMenu. This means that ToolbarDropdownMenu has the same API as
-	// DrpodownMenu.
+	// all props to the toggle of DropdownMenu. This means that ToolbarDropdownMenu
+	// has the same API as DrpodownMenu.
 	return (
-		<ToolbarItem>
+		<ToolbarItem ref={ ref } { ...props.toggleProps }>
 			{ ( toolbarItemProps ) => (
 				<DropdownMenu
 					{ ...props }
-					toggleProps={
-						props.toggleProps
-							? {
-									...props.toggleProps,
-									...toolbarItemProps,
-							  }
-							: toolbarItemProps
-					}
+					popoverProps={ {
+						...props.popoverProps,
+						isAlternate: true,
```

Comment:
Should we allow setting it to `false` at all?

Reply:
Yeah! That makes sense!